src/relay/blockchain/node.py

```diff
@@ -100,3 +103,20 @@ def send_ether(self, address):
 
     def get_block_timestamp(self, block_number):
         return self._web3.eth.getBlock(block_number).timestamp
+
+    def get_transaction_status(self, hash):
+        try:
+            # This will only find mined transactions and not pending ones
+            status = self._web3.eth.getTransactionReceipt(hash)["status"]
+            if status is not None:
```

Comment:
this should not work, as it will not be set.
I would remove this check, as status should now always be set I think, no?